proto/ibc/lightclients/groth16/v1/groth16.proto

```diff
@@ -2,23 +2,20 @@ syntax = "proto3";
 
 package celestia.ibc.lightclients.groth16.v1;
 
-option go_package = "github.com/celestiaorg/celestia-zkevm-ibc-demo/ibc/lightclients/groth16";
+option go_package = "ibc/lightclients/groth16";
```

Comment:
note to self, this file was moved which may have fixed https://github.com/celestiaorg/celestia-zkevm-ibc-demo/issues/20 although `make run-demo` no longer exists. And `make proto-gen` fails on this PR so unclear

Reply:
Good point, let me update the docs. Generation should work by just running buf generate